TradeItIosTicketSDK2/TradeItPortfolioAccountDetailsTableViewManager.swift

```diff
@@ -135,7 +153,41 @@ class TradeItPortfolioAccountDetailsTableViewManager: NSObject, UITableViewDeleg
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         if indexPath.section == SECTIONS.accountDetails.rawValue {
-            return accountDetailsCell(forTableView: tableView)
+            guard let account = self.account,
+                let accountDetail = self.accountDetails[safe: indexPath.row]
+                else { return UITableViewCell() }
+            let presenter = TradeItPortfolioBalanceEquityPresenter(account)
+
+            switch accountDetail {
+            case .totalValue:
+                return self.totalValueCell(forTableView: tableView)
+            case .totalReturn:
+                return self.accountDetailCell(
+                    forTableView: tableView,
+                    title: "Total return",
+                    value: presenter.getFormattedTotalReturnValueWithPercentage(),
+                    valueColor: presenter.getTotalReturnChangeColor()
+                )
+            case .dayReturn:
+                return self.accountDetailCell(
+                    forTableView: tableView,
+                    title: "Day return",
+                    value: presenter.getFormattedDayReturnWithPercentage(),
+                    valueColor: presenter.getDayReturnChangeColor()
+                )
+            case .buyingPower:
+                return self.accountDetailCell(
+                    forTableView: tableView,
+                    title: "Buying power",
```

Comment:
Is it possible to use the buyingPowerLabel returned by the server ? (It could be different for CIMB)
There should be an existing method on the presenter.